programs/manifest/src/state/constants.rs

```diff
@@ -20,3 +20,7 @@ pub const NO_EXPIRATION_LAST_VALID_SLOT: u32 = 0;
 
 pub const MARKET_FIXED_DISCRIMINANT: u64 = 4859840929024028656;
 pub const GLOBAL_FIXED_DISCRIMINANT: u64 = 10787423733276977665;
+
+// Amount of gas deposited for every global order. This is done to as an
+// economic disincentive to spam.
```

Comment:
Elaborate with how the mechanism works e2e

Reply:
Implemented so far:
Every time you place a global order, you deposit 5000 lamports into the global account. This is an overestimate for the gas burden on whoever will remove it from orderbook.
When you remove an order because you fill it, you cancel it yourself, you try to match and the funds for it dont exist, or you remove it because it is expired, you get the 5000 lamports.

Planned:
Separate function that 3rd party can call to do a cleanup and receive the 5000 lamports per order they remove that does not have the funds to back it or is expired.
Because swappers do not include the system program, they cant claim the 5000 when filling or cleaning. These deposits become reclaimable by the maker on their next order placement (thats just the simplest place to put it and it isnt worth a new instruction).

Reply:
I can add to the code comments if you think that would be helpful

Reply:
Think it is helpful to add to code